loader/CMakeLists.txt

```diff
@@ -16,6 +16,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 # ~~~
+include(CheckIncludeFile) # check_include_file
```

Comment:
For a second I thought this module wasn't used. Then I realized it is used in this file.

So I moved it here to make it more apparent this module is being used.